app/views/events/_card.html.slim

```diff
@@ -1,9 +1,27 @@
 .col-md-6
   = link_to event_path(event) do
     .card
-      img src="#{event.hero_image_url}"
+      .cover
+        .cover-content
+          img src="#{event.hero_image_url}"
       .card-block
         h4.card-title
           = event.name
-        p.card-text
-          | Some quick example text to build on the card title and make up the bulk of the card's content. 
+        .card-text
+          .col-sm-9
+            .min-price
+              | From 
+              span.price
+                = formated_price(event.min_price)
+            p.details
+              span.region
+                = event.region_name
+              = fa_icon "hashtag"
+              span.category
+                = event.category.name
```

Comment:
Introduce `event.category_name`. `category_name` is also best implemented with delegate.